src/main/java/com/salesforce/dynamodbv2/mt/mappers/CreateTableRequestBuilder.java

```diff
@@ -42,7 +44,7 @@ public static CreateTableRequestBuilder builder() {
 
     public CreateTableRequest build() {
         setDefaults();
-        return createTableRequest;
+        return createTableRequest.clone();
```

Comment:
`CreateTableRequestBuilder` is just there for convenience and also because I love builders.  I added some methods to it.